1-JavaScript/Week3/homework/ex3-tellFortune.js

```diff
@@ -27,30 +27,56 @@ body, this code is now written once only in a separated function.
 -----------------------------------------------------------------------------*/
 const numKids = [
   // TODO add elements here
+  0,
+  2,
+  4,
+  5,
+  8,
```

Comment:
Eight children?! How will you manage!

Reply:
Thank you Jasper for reply;

I put a random numbers of children ,yes